os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -440,11 +442,25 @@ static void _up_assert(int errorcode)
 		request_msg.cmd = BINMGR_FAULT;
 		request_msg.pid = getpid();
 
+		if (current_regs) {
+			tcb = sched_self();
+			sched_removereadytorun(tcb);
+			tcb->timeslice = 0;
```

Comment:
Please add CONFIG_RR_INTERVAL check for timeslice variable.

Reply:
done